.github/workflows/mac-tests.yml

```diff
@@ -57,15 +56,22 @@ jobs:
       run: |
         pip freeze --all
 
-    - name: Tests
+    - name: Scheduled tests
+      if:  ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
       run: |
         pytest tests -m "not integration"
 
+    - name: Tests
+      if:  ${{ github.event_name != 'schedule' && github.event_name != 'workflow_dispatch' }}
+      run: |
+        pytest tests -m "not integration and not slow"
+
+
   tests-integration-mac:
     runs-on: macos-latest
 
     # Not intended for forks.
-    if: github.repository == 'optuna/optuna'
+    if: github.repository == 'optuna/optuna' && ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
```

Comment:
This type of condition in [`tests-integration.yml`](https://github.com/optuna/optuna/blob/master/.github/workflows/tests-integration.yml#L18) is `(A && B) || B`. Should we use the same condition?

Reply:
And, this should be applied to the `tests-mac` job.